mlserver/rest/gzip_request.py

```diff
@@ -0,0 +1,25 @@
+import gzip
+from typing import Callable
+from fastapi import Request, Response
+from fastapi.routing import APIRoute
+
+
+class GzipRequest(Request):
+    async def body(self) -> bytes:
+        if not hasattr(self, "_body"):
+            body = await super().body()
+            if "gzip" in self.headers.getlist("Content-Encoding"):
+                body = gzip.decompress(body)
+            self._body = body
+        return self._body
```

Comment:
We already have a custom `Request` object defined in https://github.com/SeldonIO/MLServer/blob/master/mlserver/rest/requests.py. Would it be possible to add the `body` method there, and remove the need for the `GzipRoute` router?

Reply:
I will check that, if that will work I will make a new pull request.

Reply:
I create gzip decompress in Custom Request class (https://github.com/SeldonIO/MLServer/blob/master/mlserver/rest/requests.py.)
I will make new pull requests. I close this one